.github/workflows/pr-check.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: PR check
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Clone source code
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - name: Get VERSION
+      id: version
+      run: |
+        echo "##[set-output name=version;]$(cat VERSION)"
+    - name: Docker build
+      run: |
+        echo "Building image version ${IMAGE_VERSION}"
```

Comment:
do we need to use VERSION at all there ?

Reply:
It's good for debugging if something goes wrong, we can check the output easily -- for example if someone submits a PR but doesn't set the VERSION correctly.

Reply:
I don't see where it checks that VERSION is correct ?

I mean, if I populate VERSION file with `hello world` I don't see any checks

Reply:
By check I mean verify by looking at the output, if someone were to put `hello world` in `VERSION` then the PR check would fail, and just checking the build output would show the cause of the failure.

Reply:
Okay I'll remove it.

Reply:
you can keep it if you really check for example that VERSION is a valid content